api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -88,6 +97,14 @@ public Workspace apply(org.pmiops.workbench.db.model.Workspace workspace) {
           if (workspace.getCdrVersion() != null) {
             result.setCdrVersionId(String.valueOf(workspace.getCdrVersion().getCdrVersionId()));
           }
+
+          ArrayList<org.pmiops.workbench.db.model.WorkspaceUserRole> usersOnWorkspace = new ArrayList<org.pmiops.workbench.db.model.WorkspaceUserRole>();
+          Iterator<org.pmiops.workbench.db.model.WorkspaceUserRole> iter = workspace.getWorkspaceUserRoles().iterator();
+          while(iter.hasNext()) {
+            usersOnWorkspace.add(iter.next());
+          }
+          result.setUserRoles(usersOnWorkspace.stream().map(TO_CLIENT_USER_ROLE).collect(Collectors.toList()));
```

Comment:
`Collection` actually exposes `stream()`, and `Set` is a `Collection`, so I think you can just call `getWorkspaceUserRoles().stream()`.